plugins/com.aptana.editor.common/src/com/aptana/editor/common/AbstractThemeableEditor.java

```diff
@@ -1334,7 +1334,7 @@ protected IVerticalRulerColumn createLineNumberRulerColumn()
 		return fLineNumberRulerColumn;
 	}
 
-	private boolean isWordWrapEnabled()
+	private boolean isAptanaWordWrapEnabled()
```

Comment:
Any specific reason for this?

Reply:
With the new version of `org.eclipse.ui.workbench.texteditor` plugin, there exists a method with name `isWordWrapEnabled()` now, and that method marked as `final`. Renamed the local method to `isAptanaWordWrapEnabled()` so that, we can still achieve what we were doing(fetching the flag from preferences).